src/w3d/renderer/render2dsentence.cpp

```diff
@@ -226,10 +226,10 @@ void FontCharsClass::Create_GDI_Font(const char *font_name)
     }
 
     if (is_generals) {
-        font_width = ((double)-font_height / 2.5);
+        font_width = ((float)-font_height * 0.4f);
```

Comment:
/ 2.5 is the same thing

Reply:
I know. Do you want to keep the `2.5` ? World Builder shows * 0.4f